.travis.yml

```diff
@@ -1,20 +1,16 @@
 language:
```

Comment:
Can you remove this "language" section? It is overridden by the node one, and was only there to begin with because that was how you used to specify osx in Travis.